apps/src/code-studio/pd/workshop_dashboard/components/survey_results/matrix_choice_responses.jsx

```diff
@@ -45,7 +46,7 @@ export default class MatrixChoiceResponses extends React.Component {
               innerAnswer = answer[innerQuestionId];
             }
 
-            if (!innerAnswer) {
+            if (!innerAnswer || _.isEmpty(innerAnswer)) {
```

Comment:
You found a bug here?

Reply:
Yeah the facilitator innerAnswer could be empty instead of null, which caused a question to show up with no answer data